### PR TITLE
OOP school library: unit tests

### PR DESCRIPTION
**Pull request by Laylo309**
- Add unit tests to classes:
  - `Book`
  - `Person`
  - `Student`
  - `Teacher`
  - `Classroom`
  - `Corrector`
  - `Rental`

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Laylo309 & @nelsinofrancisco, 

**This is a great project you have going on, well-done lads**. The tests cover a wide scenario base and cater to all the class methods.

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] Clean up the getting started section in the readme file. It is currently describing how Ruby can be used to decode a Morse code message.
- [ ] Add the necessary step-by-step instructions to follow to set up your project locally. Remember to add all the necessary commands to run, including the test command. A readme should always be professional.



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -9,17 +9,24 @@
...
-This repository includes files with Ruby that can be used to decode a Morse code message:
+This repository includes files with Ruby that can be used to is getting information from the library about the University:
 
 - Please run the program in a IRB enviorement
-
-## Author
+- From your terminal you have to run the following command: `ruby main.rb`
+- After running this command you will get following kind of information:
+![screenshot](./Screenshot_2.png)
+  ## Author
```
Comment (hiromataba) [inline README.md]: - [ ] [OPTIONAL]You did great :+1: adding how to run your project but as said by the previous reviewer you should also add how to get your project locally 

- Here is a hint of how you may do that :point_down: 
image

File: README.md [hunk 2]
```diff
@@ -9,17 +9,24 @@
...
 
-This repository includes files with Ruby that can be used to decode a Morse code message:
+This repository includes files with Ruby that can be used to is getting information from the library about the University:
 
 - Please run the program in a IRB enviorement
-
-## Author
+- From your terminal you have to run the following command: `ruby main.rb`
+- After running this command you will get following kind of information:
+![screenshot](./Screenshot_2.png)
```
Comment (hiromataba) [inline README.md:23]: - As said by the previous reviewer you should also add how to run test, you can write something like this :point_down: 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Laylo309 and @nelsinofrancisco  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

required1

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Laylo309) [reply README.md]: Done ✔

File: README.md (hunk 2 above)
Comment (Laylo309) [reply README.md:23]: Done ✔

**Review by EricMbouwe — APPROVED** [review]
### Hi @Laylo309, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**